### DISCUSSION TITLE
`Conversation` and `Conversations` are incredibly confusing

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#18049] [Pull Requests]
### TL;DR

`Conversation` and `Conversations` are incredibly confusing -- that `s` is doing way too much work.

### Let's look at three PRs

https://github.com/github/gh-gei/pull/435/files
<img width="764" alt="No conversations" src="https://user-images.githubusercontent.com/2119212/172450347-ed7772e2-cd8d-4efb-a05f-3c7ec06ac1df.png">

Maybe this jump to conversation can jump to the conversation tab?

https://github.com/github/gh-gei/pull/433/files
<img width="761" alt="one conversation with 3 items but no conversations" src="https://user-images.githubusercontent.com/2119212/172450491-14e89dd8-8d7f-4a3c-859d-c9d7e8733020.png">

Oh, apparently not, because there is a conversation with 3 items and there are no conversations, and I can't jump to the conversation.

https://github.com/github/gh-gei/pull/421/files
![one conversation with 7 and 4 unresolved conversations](https://user-images.githubusercontent.com/2119212/172450595-5ecc60bf-567b-4acd-a142-a37ff3d0b908.png)

Clearly these things are mostly unrelated, except in fact, they aren't, because the conversation tab has 8 items - the description, 3 comments, and 4 items that are cross linked from the unresolved conversations view.

### Thoughts

I, personally, understand how they're related, but a colleague rightfully complained that this is incredibly confusing. Calling both of these things a `conversation` and only making a distinction via an `s` is really unfortunate.
